### PR TITLE
sentinel updates and add tests

### PR DESCRIPTION
**Pull request by dev-mlb**

### CODE REVIEW

File: src/main/java/emissary/core/sentinel/Sentinel.java [hunk 1]
```diff
@@ -285,17 +277,22 @@ public void incrementTimer(long time) {
...
-            return new StringJoiner(", ", "[", "]")
-                    .add("agentName='" + agentName + "'")
-                    .add("placeName='" + placeName + "'")
-                    .add("shortName='" + shortName + "'")
-                    .add("timer=" + timer + " minute(s)")
+            return new StringJoiner(", ", "{", "}")
+                    .add("\"agentName\":\"" + agentName + "\"")
+                    .add("\"directoryEntry\":\"" + directoryEntryKey + "\"")
+                    .add("\"shortName\":\"" + shortName + "\"")
+                    .add("\"timeInMinutes\":" + timer)
```
Comment (jpdahlke) [inline src/main/java/emissary/core/sentinel/Sentinel.java:295]: confirming you intend this to print like: `"timeInMinutes":2` and not `"timeInMinutes":"2"`

File: src/main/java/emissary/core/sentinel/protocols/Protocol.java [hunk 2]
```diff
@@ -23,7 +23,7 @@
 import java.util.StringJoiner;
 import java.util.concurrent.ConcurrentHashMap;
 
-import static emissary.core.sentinel.Sentinel.Tracker.getPlaceSimpleName;
+import static emissary.core.sentinel.Sentinel.Tracker.getPlaceName;
```
Comment (jpdahlke) [inline src/main/java/emissary/core/sentinel/protocols/Protocol.java:26]: Not needed?

**Review by jpdahlke — CHANGES_REQUESTED** [review]
Change look good, just minor nits.

File: src/main/java/emissary/core/sentinel/protocols/Protocol.java (hunk 2 above)
Comment (dev-mlb) [reply src/main/java/emissary/core/sentinel/protocols/Protocol.java]: Nuked

**Review by jpdahlke — APPROVED** [review]

**Review by cfkoehler — APPROVED** [review]

**Pull request merged**